.ruby-version

```diff
@@ -1 +1 @@
-2.3.1
+ruby-2.4.1
```

Comment:
re: what is allowed in the `.ruby-version` file

Short answer:  `.rbenv` expects only the _version number_ in the file.  "ruby-" can't be in there.

### tldr:
rbenv expects the contents to match the 'name' of one of the ruby versions that it installs.   [This is the list of 'names' of possible versions that can be installed](https://github.com/rbenv/ruby-build/tree/master/share/ruby-build)
(tldr: the list is provided via `ruby-build`).